microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
 
 When DevTools is used to debug processes that are running in Node.js, this experiment makes whatever `console.profile` generates available to the **Performance** Tool.
 
+See also:
+* [JavaScript Profiler tool](../javascript-profiler/javascript-profiler-tool.md)
+
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
```

Comment:
```suggestion
## Timeline: Enable track configuration feature that can reorder or hide a track in the flame chart
```

Reply:
confirmed in UI (d/k why grayed out, i think we discussed or logged bug):
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/842ed439-ed5a-4cbb-91de-cc481af23a66)